backend/core/src/seeds/listings/shared.ts

```diff
@@ -239,8 +239,10 @@ export const defaultListing: ListingSeedType = {
 }
 
 // Preferences
-export function getLiveWorkPreference() {
-  return JSON.parse(JSON.stringify(liveWorkPreference))
+export function getLiveWorkPreference(jurisdictionName) {
```

Comment:
NBD but seems like this and the duplicates below could be getPreference(preference, jurisdiction)